components/gfx/display_list/mod.rs

```diff
@@ -423,7 +423,11 @@ impl BaseDisplayItem {
                 node: OpaqueNode(0),
                 pointing: None,
             },
-            local_clip: LocalClip::from(max_rect().to_rectf()),
+            // Create a rectangle of maximal size.
+            local_clip: LocalClip::from(LayoutRect::new(
```

Comment:
It probably makes sense to implement max_rect for LayoutRect if we can. Perhaps this can be a trait.

Reply:
There is a trait [`MaxRect`](https://doc.servo.org/webrender/util/trait.MaxRect.html) in webrender but I can't use it from servo.

Maybe this trait should instead live in euclid?

Reply:
I think it makes sense to at least copy the trait from WebRender and replace max_rect with it.